src/store/user-store.js

```diff
@@ -11,6 +13,18 @@ class UserStore {
   setIsLoggedIn(value) {
     this.isLoggedIn = value;
   }
+
+  setUserResponseData(data) {
+    this.userResponse = data;
+  }
+
+  get userProfileImageUrl() {
+    if (this.userResponse.images) {
+      return this.userResponse?.images[0]?.url;
+    }
+
+    return "";
+  }
```

Comment:
Step 6: Now that we have the data set up and populating in the app (test it out by running console logs), we can now use a computed value to get the users profile picture.

You can see how we have `?` on line 23. These are called optional chaining and we will go over it more later on but if you want to get ahead and read up on it, feel free to take a look at [this documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Reply:
As soon I was about to ask a question about the "?" I saw the documentation link. Read my mind.